azurerm/internal/services/trafficmanager/resource_arm_traffic_manager_profile.go

```diff
@@ -106,6 +106,25 @@ func resourceArmTrafficManagerProfile() *schema.Resource {
 							},
 						},
 
+						"custom_headers": {
+							Type:     schema.TypeList,
+							Optional: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"name": {
+										Type:             schema.TypeString,
+										Required:         true,
+										ValidateFunc:     validation.StringIsNotEmpty,
+										DiffSuppressFunc: suppress.CaseDifference,
```

Comment:
What is the reason for ignoring case here?

Reply:
I believe [per standard][rfc-7230] that header names are case-insensitive. Not sure if there's a newer standard, please let me know if there's a reason this should be case sensitive :slightly_smiling_face: 

[rfc-7230]:https://tools.ietf.org/html/rfc7230#section-3.2